### ISSUE TITLE
some confuses about learning rate 

### ISSUE THREAD

**Issue by EternalEvan** [argusswift/YOLOv4-pytorch#91]
I'm training with the coco2017 dataset. When the training reaches the 26th epoch, the displayed LR = 0.0000

[2020-11-17 12:49:56,134]-[train.py line:223]:  === Epoch:[ 26/50],step:[42640/44640],img_size:[416],total_loss:50.3763|loss_ciou:11.0461|loss_conf:15.9487|loss_cls:23.3814|lr:0.0000 

Is it because of the decay of learning rate, which leads to some print bugs,  or other errors?

**Comment by jingtianyilong** [comment]
`{:06f}`would help

**Comment by EternalEvan** [comment]
Thank you very much ！！！
